DOCKER.md

```diff
@@ -4,73 +4,66 @@ Docker Image of iot-lab-gateway
 The included Dockerfile includes all the necessary dependencies listed in `INSTALL.md`, without you having
 to install them on your host machine.
 
-The only prerequisite before running the image for the first time is installing udev rules on the host
-
-    python setup.py udev_rules_install
-
 For the following commands sudo might be needed depending on whether your user is in the `docker` group:
 
 To build the image :
 
     docker build -t iot-lab-gateway .
 
-To run the image:
+To run the image, assuming the open node is plugged and assigned to /dev/ttyUSB0:
 
-    ./docker-run
+    ./docker-run /dev/ttyUSB0
 
 Command line arguments for docker-run:
 
     usage: docker-run [--help] [-v VOLUME] [-b BOARD_TYPE] [-h HOSTNAME]
-                      [-c NODE_TYPE] [-d] [-r]
-                      [cmd [cmd ...]]
+                      [-c CONTROL_NODE_TYPE] [-d] [-r] [--cntty CNTTY]
+                      ontty
 
     positional arguments:
-      cmd                   Command to run inside the docker container (default:
-                            None)
+      ontty                 Open Node tty to pass inside the docker container
 
     optional arguments:
       --help
       -v VOLUME, --volume VOLUME
-                            Host directory containing the gateway code, usually gateway_code (default: None)
+                            Host directory containing gateway_code (default: )
       -b BOARD_TYPE, --board-type BOARD_TYPE
                             Set node board type (default: m3)
       -h HOSTNAME, --hostname HOSTNAME
                             Set hostname (default: custom-123)
-      -c NODE_TYPE, --node_type NODE_TYPE
-                            Set control_node_type (default: no)
-
-    options:
-      -d, --daemon          Daemon mode
-      -r, --reloader        Reloader
+      -c CONTROL_NODE_TYPE, --control-node-type CONTROL_NODE_TYPE
+                            Set node as control node (default: no)
+      -d, --daemon          Daemon mode (default: False)
+      -r, --reloader        Reloader (default: False)
+      --cntty CNTTY         Control Node tty to pass inside the docker container
 
 
 You can mount a gateway_code folder into the docker container, so that you can modify your code, and have it used inside the container
 directly, usually:
 
-    ./docker-run -v gateway_code
+    ./docker-run -v gateway_code /dev/ttyUSB0
 
 ## Examples
 
 You can use any type of open node:
 
-    ./docker-run -b samr21 -h samr21-test1
+    ./docker-run -b samr21 -h samr21-test1 /dev/ttyUSB0
```

Comment:
Here it should be /dev/ttyACM0 (for samr21)

Reply:
OK, fixed the doc